### DISCUSSION TITLE
Vanguard

### DISCUSSION THREAD

**Discussion by mtttz3q** [Flowseal/zapret-discord-youtube#971] [Q&A]
Мужики, прошу без хейта, не шарю за всю тему, появилась инфа о том, что vanguard откидывает пермачом за обход ютуба и дискорда, возможно ли это вообще и стоил ли переживать?

**Comment by iromka** [comment]
Привет. Не знаю как райот геймс (убрать лишнюю букву) себя ведут, но, например, та же кода не дает запустить себя когда включен запрет/гудбайдпи. Вальвам пофиг, фейситу тоже. 
Так что тут надо смотреть насколько глинят конкретные разрабы.

**Reply by mtttz3q** [reply]
а колда на каком античите работает?

**Comment by Grafell** [comment]
Играю с активным запретом в лол с начала блокировок ютуба. Пермача нет)
